### DISCUSSION TITLE
Motrix 种子文件无法下载 求帮忙🙏

### DISCUSSION THREAD

**Discussion by yvonne66999** [agalwood/Motrix#1220] [Q&A]
Motrix Version 1.6.11，电脑是macOS Monterey 12.13.1。下载界面显示0kb/s，右下角也是0kb/s。网络ip测试是没有接入IPV6，请问要怎么解决呢？

**Comment by cccasqq** [comment]
需要先下载一些热种，后面会有速度，具体的速度是看分享的热度，热度高的速度快，没啥的热度的没有速度。

**Reply by yvonne66999** [reply]
下载什么速度都是0😭但是用浏览器直接下载东西都是正常的

**Comment by mao-netizen** [comment]
啊，同样的问题，解决了吗😩

**Comment by yvonne66999** [comment]
emmm。没有解决 换了其他软件也是一样的情况 我觉得可能是种子太老了。。。。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月8日(星期一) 晚上7:08
收件人: ***@***.***&gt;; 
抄送: "Lin ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [agalwood/Motrix] Motrix 种子文件无法下载 求帮忙🙏 (Discussion #1220)





 
啊，同样的问题，解决了吗😩
